plugins/modules/ome_alert_policies.py

```diff
@@ -0,0 +1,1120 @@
+            type: str
+requirements:
+    - "python >= 3.9.6"
+author: "Jagadeesh N V(@jagadeeshnv)"
+notes:
+    - Run this module from a system that has direct access to Dell OpenManage Enterprise
+      or OpenManage Enterprise Modular.
+    - This module supports IPv4 and IPv6 addresses.
+    - This module supports C(check_mode).
+'''
+
+EXAMPLES = r'''
+---
+- name: "Create an alert policy"
+  dellemc.openamanage.ome_alert_policies:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    name: "Alert Policy One"
+    device_service_tag:
+      - ABCD123
+      - SVC7845
+    category:
+      - catalog_name: Application
+        catalog_category:
+          - category_name: Audit
+            sub_category_names:
+              - Generic
+              - Devices
+      - catalog_name: iDRAC
+        catalog_category:
+          - category_name: Audit
+            sub_category_names:
+              - BIOS Management
+              - iDRAC Service Module
+    date_and_time:
+      date_from: "2023-10-10"
+      date_to: "2023-10-11"
+      time_from: "11:00"
+      time_to: "12:00"
+    severity:
+      - unknown
+      - critical
+    actions:
+      - action_name: Trap
+        parameters:
+          - name: "192.1.2.3:162"
+            value: true
+          - name: "traphostname.domain.com:162"
+            value: true
+  tags: create_alert_policy
+
+- name: "Update an alert Policy"
+  dellemc.openamanage.ome_alert_policies:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    new_name: "Update Policy Name"
+    device_group: "Group Name"
+    message_ids:
+      - AMP400
+      - CTL201
+      - BIOS101
+    date_and_time:
+      date_from: "2023-10-10"
+      date_to: "2023-10-11"
+      time_from: "11:00"
+      time_to: "12:00"
+      time_interval: true
+    actions:
+      - action_name: Trap
+        parameters:
+          - name: "192.1.2.3:162"
+            value: true
+  tags: update_alert_policy
+
+- name: "Enable an alert policy"
+  dellemc.openamanage.ome_alert_policies:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    name: "Policy Name"
+    enable: true
+  tags: enable_alert_policy
+
+- name: "Disable multiple alert policies"
+  dellemc.openamanage.ome_alert_policies:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    name:
+      - "Policy Name 1"
+      - "Policy Name 2"
+    enable: false
+  tags: disable_alert_policy
+
+- name: "Delete an alert policy"
+  dellemc.openamanage.ome_alert_policies:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    name:
+      - "Policy Name"
+    state: absent
+  tags: delete_alert_policy
+'''
+
+RETURN = r'''
+---
+msg:
```

Comment:
should we add in this section for "changed" as well?

Reply:
yes msg applies to all scenarios, this key is returned always